obsah/__init__.py

```diff
@@ -99,7 +99,12 @@ def takes_target_parameter(self):
         with open(self.path) as playbook_file:
             plays = yaml.safe_load(playbook_file.read())
 
-        return any(self.application_config.target_name() in play.get('hosts', []) for play in plays)
+        if not isinstance(self.application_config.target_name(), list):
+            target_names = set([self.application_config.target_name()])
+        else:
+            target_names = set(self.application_config.target_name())
```

Comment:
I think we can safely just change the interface of https://github.com/theforeman/obsah/blob/43d9bfd053eed827b1468e4b54c24c7e471e9245/obsah/__init__.py#L179-L184 instead of the `if/else`?

Reply:
I am fine with that, I was just erring on the side of not breaking the API. Would we change this and then release a 0.4?

Reply:
https://github.com/theforeman/obsah/pull/28 is my attempt to do it